### PR TITLE
Added new module DAEMS-001

### PR DESCRIPTION
**Pull request by driesdaems10**
Added template for new module 2022-DAEMS-001

### CODE REVIEW

File: 2022-DAEMS-001/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) <YEAR> <COPYRIGHT HOLDER>
```
Comment (LimesLimits) [inline 2022-DAEMS-001/LICENSE:3]: you forgot to fill this in

File: 2022-DAEMS-001/NASSA.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - variable initialisation
+  - agent variables
+  - agent behaviour (self)
+programmingKeywords:
+  - Object-oriented
+implementations:
+  - language: NetLogo
+    codeDir: netlogo_implementation/
+    softwareDependencies:
+      - language/software version 6.2.2
```
Comment (LimesLimits) [inline 2022-DAEMS-001/NASSA.yml:35]: please specify software name

File: 2022-DAEMS-001/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# 2022-DAEMS-001
+
+*by Dries Daems* (NASSA submission :rocket:)
+
+Description of what the module does. It should expand the information already given in the name/title.
```
Comment (LimesLimits) [inline 2022-DAEMS-001/README.md:5]: please delete automated text

File: 2022-DAEMS-001/documentation/designDetails.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# Design details
```
Comment (LimesLimits) [inline 2022-DAEMS-001/documentation/designDetails.md:1]: please delete empty files

File: 2022-DAEMS-001/netlogo_implementation/resourceExp.nlogo [hunk 5]
```diff
@@ -0,0 +1,497 @@
...
+  storage ; one of the patches is turned into a homebase where the rangers keep their stock as storage
+]
+
+rangers-own [
+  stock ;rangers can carry resources from patches to their homebase as stock
+]
+
+to setup
+  ca
+  create-rangers 1 [
```
Comment (LimesLimits) [inline 2022-DAEMS-001/netlogo_implementation/resourceExp.nlogo:16]: get rid of the magic numbers

File: 2022-DAEMS-001/netlogo_implementation/resourceExp.nlogo [hunk 6]
```diff
@@ -0,0 +1,497 @@
...
+  tick
+end
+
+to exploit-resources
+  ask rangers [
+    let target patches with-max [resource] ; agent looks for patches with highest resources
+    move-to one-of target ; agent moves to one of the patches with the highest resources
+    set stock stock + resource
+    ask patch-here [
+      set resource 0
```
Comment (LimesLimits) [inline 2022-DAEMS-001/netlogo_implementation/resourceExp.nlogo:39]: would be nice to update patch colour as well

**Review by LimesLimits — CHANGES_REQUESTED** [review]

File: 2022-DAEMS-001/netlogo_implementation/resourceExp.nlogo (hunk 5 above)
Comment (driesdaems10) [reply 2022-DAEMS-001/netlogo_implementation/resourceExp.nlogo:16]: Thank you for pointing this out.

File: 2022-DAEMS-001/netlogo_implementation/resourceExp.nlogo (hunk 6 above)
Comment (driesdaems10) [reply 2022-DAEMS-001/netlogo_implementation/resourceExp.nlogo:39]: Agreed, but it is not necessary for the purpose of the module. So for the sake of simplicity I wouls prefer to leave it as it is.

File: 2022-DAEMS-001/LICENSE (hunk 1 above)
Comment (driesdaems10) [reply 2022-DAEMS-001/LICENSE:3]: Done

File: 2022-DAEMS-001/NASSA.yml (hunk 2 above)
Comment (driesdaems10) [reply 2022-DAEMS-001/NASSA.yml:35]: Done

File: 2022-DAEMS-001/documentation/designDetails.md (hunk 4 above)
Comment (driesdaems10) [reply 2022-DAEMS-001/documentation/designDetails.md:1]: Done

File: 2022-DAEMS-001/netlogo_implementation/resourceExp.nlogo (hunk 5 above)
Comment (driesdaems10) [reply 2022-DAEMS-001/netlogo_implementation/resourceExp.nlogo:16]: Done

**Pull request merged**